### ISSUE TITLE
Collection Description component not rendering after Beta.79 migration

### ISSUE THREAD

**Issue by 58bits** [payloadcms/payload#7784]
### Payload Version

3.0.0-beta.88

### Node Version

20

### Next.js Version

15.0.0-canary.104

### Describe the Bug

Prior to beta.79 - we could place a custom collection description  component in the following position:

```ts
admin: {
    enableRichTextLink: false,
    enableRichTextRelationship: false,
    group: 'Media and Uploads',
    components: {
      edit: {
        Description: MediaCollectionDescription
      }
    }
  },
  ```
  
During the migration to >= beta.79 - we updated the above to use a source path for the component...

```ts
admin: {
    enableRichTextLink: false,
    enableRichTextRelationship: false,
    group: 'Media and Uploads',
    components: {
      edit: {
        Description:
          '/_payload/collections/uploads/media/components/media-collection-description#MediaCollectionDescription'
      }
    }
  },
```

... however, after migration the edit.Description component no longer renders.

The Description component  still appears in Payload types....

![image](https://github.com/user-attachments/assets/897d5b12-6f32-4db0-9d82-91d4a28ae1a9)

...although edit.Description no longer appears in the beta docs for admin components -  https://payloadcms.com/docs/beta/admin/collections#components

If this option has been removed, could it be documented in the release notes? And could the docs be updated to describe an alternative method for creating a custom collection description component? One that appears below the collection label, but before the table view? As in....

![image](https://github.com/user-attachments/assets/fc095bda-b883-4b2f-bdac-727d3a8ef535)

We've tried using the admin.description option - which works for strings (or translated strings) but does not appear to work for React client components - or at least it's unclear how to do so.

Hope this helps.

### Reproduction Steps

As above.

**Comment by jacobsfletch** [comment]
@58bits this has been fixed in #7789. One thing to note here is that this component should actually have been defined on the `admin.components.Description` key, _not_ the `admin.components.edit.Description`. Check out that PR for full details.

**Comment by 58bits** [comment]
@jacobsfletch awesome - thank you!
